test/storage/storage_test.dart

```diff
@@ -983,7 +984,8 @@ void main() {
         var bucket = api.bucket(bucketName);
         bucket.info(objectName).then(expectAsync1((stat) {
           expect(stat.name, objectName);
-          expect(stat.updated, DateTime(2014));
+          expect(
+              stat.updated, anyOf(DateTime(2014), DateTime.utc(2014, 1, 1, 8)));
```

Comment:
Why 8?

Probably you just leaked your own timezone, :rofl: , which is why it fails in CI.

Perhaps we just need to make the mocked server respond with `..updated = DateTime.utc(2014)` ?